README.md

```diff
@@ -71,9 +80,9 @@ Belle is friends with Mrs. Potts
 ```
 
 Write your code here:
-```ruby
-# code here
-```
+@friends.each do |friends|
+puts "Belle is friends with #{@friends}"
```

Comment:
You want to reference the iteration variable (i.e., the thing in the pipes) - not the collection itself - inside of the each statement. The iteration variable represents each individual item in the array.

In the end, it should look something like...

```rb
friends.each do |friend|
  puts "Belle is friends with #{friend}"
end
```